src/cloud/release-notes/mcd-release-notes.md

```diff
@@ -16,6 +16,30 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v1.2.1
+*Release date: December 21, 2020*<br/>
+
+-  {:.new}<!--MCLOUD-7259-->**NGINX command options.** Added build command options to change the number of NGINX `worker_processes` and NGINX `worker_connections` for TLS and Web services. The `worker_process` parameter retains the ability to set the value to `auto`. Examples:
+
+    ```bash
+    ./vendor/bin/ece-docker build:compose --nginx-worker-processes=2
```

Comment:
So, placing two commands in a bash code block makes the copy command ineffective for copying a single line. Either use two separate code blocks, or use a different determiner than bash so that it's more like a list of commands than a command to copy and run.